.gitignore

```diff
@@ -13,3 +13,6 @@ palette.pdf
 README.md
 public/javascript/src
 public/javascript/src/*
+grotonfiresandbox
+.env
+variables.env.sample
```

Comment:
This should be committed to help remember what variables need to be populated for the code to run.

Reply:
Got it. Just removed `grotonfiresandbox` and `variables.env.sample` from the `.gitignore` file. `.env` will stay.